### PR TITLE
Removed setup.py file from Python SDK examples

### PR DESCRIPTION
**Pull request by akaila-splunk**
- removed setup.py file and update import statements of python examples

### CODE REVIEW

File: python/analytics/input.py [hunk 1]
```diff
@@ -18,6 +18,8 @@
 import sys
 
 from datetime import datetime
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", "..", "lib"))
```
Comment (fantavlik) [inline python/analytics/input.py:22]: This is great, this matches what app developers will need to do which makes for more accurate examples

File: python/dashboard/feed.py [hunk 2]
```diff
@@ -20,9 +20,11 @@
 import pandas as pd
 import matplotlib.pyplot as plt
 
-sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", "..", "lib"))
 from splunklib import client
 from splunklib import results
+
+sys.path.insert(0, os.path.join(os.path.dirname(__file__), "..", ".."))
```
Comment (fantavlik) [inline python/dashboard/feed.py:27]: I don't think this is needed?

**Comment by fantavlik** [comment]
I don't see any changes to actually remove setup.py - I think that's missing :)

**Review by fantavlik — APPROVED** [review]

**Comment by fantavlik** [comment]
One non-blocking note about an extra sys.path.append that may not be needed

File: python/dashboard/feed.py (hunk 2 above)
Comment (akaila-splunk) [reply python/dashboard/feed.py]: Yes. thanks for highlighting, we missed this. We have removed the extra line.

**Pull request merged**